### DISCUSSION TITLE
Send Byte audio data from call API to Whisper

### DISCUSSION THREAD

**Discussion by BakingBrains** [openai/whisper#1131] [Q&A]
I am trying to transcribe byte data coming from twilio call stream. I followed this blog, https://www.twilio.com/blog/transcribe-phone-calls-text-real-time-twilio-vosk I want to replace vosk with whisper. I only doubt I have is regarding the input byte. 

![226593213-9480b0cc-3174-4d8b-8e98-18046f9b24db](https://user-images.githubusercontent.com/51019420/226919870-8f3f740b-273e-4602-8ad8-b5f8546dc9c8.png)

I tried with load_audio method mentioned here (https://github.com/openai/whisper/discussions/380) still no use.
```
audio = load_audio(audio)
customaudio_data = np.frombuffer(audio, np.int16).flatten().astype(np.float32) / 32768.0
text, _, _ = model.transcribe(customaudio_data)
```

Any suggestions? 

Thanks and Regards

**Comment by BakingBrains** [comment]
Any suggestions here?

Thank You.

**Comment by justinlevi** [comment]
I have been working on something similar. Just got this working here; 
https://github.com/collabora/WhisperLive/issues/40
